### PR TITLE
📖 fix blog post link in quick-start on documentation

### PR DESCRIPTION
**Pull request by JulienBreux**
Hello 👋🏻 

A very tiny fix to fix the blog post link in the quick-start section on documentation

Have a good day 🚀

### CODE REVIEW

File: docs/book/src/quick-start.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ kubebuilder init --domain my.domain --repo my.domain/guestbook
 If your project is initialized within [`GOPATH`][GOPATH-golang-docs], the implicitly called `go mod init` will interpolate the module path for you.
 Otherwise `--repo=<module path>` must be set.
 
-Read the [Go modules blogpost][go-modules-blogpost] if unfamiliar with the module system.
+Read the [Go modules blogpost](https://blog.golang.org/using-go-modules) if unfamiliar with the module system.
```
Comment (Iceber) [inline docs/book/src/quick-start.md:68]: I think we can change L226
https://github.com/kubernetes-sigs/kubebuilder/blob/b6ecf5ef413852d8ca2a7b0281c54e316dbb5b29/docs/book/src/quick-start.md#L226
```
[go-module-blogpost]: https://blog.golang.org/using-go-modules
```

Comment (JulienBreux) [reply docs/book/src/quick-start.md]: Good idea! You're right! Thanks

**Comment by Iceber** [comment]
@JulienBreux The `[Go modules blogpost][go-modules-blogpost]` is not working because of the missing space in line L226, so we just need to add a space in line 226, not change line 68
https://github.com/kubernetes-sigs/kubebuilder/blob/b6ecf5ef413852d8ca2a7b0281c54e316dbb5b29/docs/book/src/quick-start.md#L225-L228

**Comment by JulienBreux** [comment]
> @JulienBreux The `[Go modules blogpost][go-modules-blogpost]` is not working because of the missing space in line L226, so we just need to add a space in line 226, not change line 68
> https://github.com/kubernetes-sigs/kubebuilder/blob/b6ecf5ef413852d8ca2a7b0281c54e316dbb5b29/docs/book/src/quick-start.md#L225-L228

OMG 😱 I'm so stupid, sorry, I need coffee ☕ 
Thanks!

File: docs/book/src/quick-start.md [hunk 2]
```diff
@@ -223,6 +223,6 @@ Now, see the [architecture concept diagram][architecture-concept-diagram] for a
 [pre-rbc-gke]: https://cloud.google.com/kubernetes-engine/docs/how-to/role-based-access-control#iam-rolebinding-bootstrap
 [cronjob-tutorial]: https://book.kubebuilder.io/cronjob-tutorial/cronjob-tutorial.html
 [GOPATH-golang-docs]: https://golang.org/doc/code.html#GOPATH
-[go-module-blogpost]:https://blog.golang.org/using-go-modules
+[go-module-blogpost]: https://blog.golang.org/using-go-modules
```
Comment (Iceber) [inline docs/book/src/quick-start.md:226]: I found that `go-module-blogpost` also needs to be modified to `go-modules-blogpost`.
https://github.com/kubernetes-sigs/kubebuilder/blob/b6ecf5ef413852d8ca2a7b0281c54e316dbb5b29/docs/book/src/quick-start.md#L68

Comment (JulienBreux) [reply docs/book/src/quick-start.md]: In the plural is better. What do you think?

Comment (Iceber) [reply docs/book/src/quick-start.md]: Yes:smiley:

**Comment by JulienBreux** [comment]
@Iceber it's good for you? 😊

**Review by camilamacedo86 — APPROVED** [review]
Thank you  for the contribution 🥇 
/lgtm

**Comment by yuswift** [comment]
The typo still exists on website. Can anybody help update the website?
see https://github.com/kubernetes-sigs/kubebuilder/issues/2240

**Comment by JulienBreux** [comment]
@camilamacedo86 hi :) did we rebuild the documentation? After this merger. 🙏🏻

**Pull request merged**